app/utils/store.py

```diff
@@ -31,7 +31,8 @@ def put(self, service_id, document_stream, *, mimetype):
             Body=document_stream,
             ContentType=mimetype,
             SSECustomerKey=encryption_key,
-            SSECustomerAlgorithm='AES256'
+            SSECustomerAlgorithm='AES256',
+            Metadata=metadata
```

Comment:
Also, just tried executing this without email validation, and it looks like `None` is not an acceptable value to actually send into the S3 call - so if `metadata = None`, we need to not add this param to the call.

Reply:
ok, is an empty hash an acceptable value?

Reply:
I pushed a version with empty hash to make it easier for you to check ☺️